packages/fxa-content-server/tests/functional/settings/recovery_key.js

```diff
@@ -304,7 +304,7 @@ registerSuite('Recovery key - unverified session', {
           .then(click(selectors.RECOVERY_KEY.GENERATE_KEY_BUTTON))
           // if the session is unverified, then the modal will be shown.
           .then(testElementExists('[data-testid=modal-verify-session]'))
-          .then(fillOutVerificationCode(email, 0))
+          .then(fillOutVerificationCode(email, 1))
```

Comment:
Add a comment why we look for index 1 now and/or also check for new device login email?